tool/tsh/common/tsh.go

```diff
@@ -1394,6 +1392,10 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 		defer runtimetrace.Stop()
 	}
 
+	if ok, err := vnetCommands.tryRun(&cf, command); ok || err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
This approach to launching a command is suppose to allow us to have different VNet subcommands per OS without forcing us to have a "not implemented" version for every other OS, right?

I think it'd be worthwhile to note this down here so that people don't copy this convention for other commands where it's not strictly necessary.

The fact that `tryRun` is not O(1) pains me a bit, but in the grand scheme of things it probably doesn't make that much of a difference, given that we only have a handful of subcommands. 🙃

Reply:
I would question whether switching subcommand by OS or context is actually simpler than just having all the subcommands there all the time (and failing if they can't do anything meaningful).